src/docs/configuration.html

```diff
@@ -41,11 +41,12 @@ <h1>Overlay Configuration API</h1>
 			<th scope="row">
 				<code><a href="#demo">DEMO</a></code>
 			</th>
-			<td><code>true</code> or <code>false</code></td>
+			<td><code>true</code>, <code>false</code> or <code>static</code></td>
```

Comment:
Should this be `static` or `STATIC`?

Reply:
lowercase `static` as that is the value the queryparam accepts (the string `?DEMO=static`), while the uppercase is the value used by JavaScript once the configuration goes through the `transform`  function in `cleanse-query-params.js`.
The difference between what is the public query param value and the internal representation of that value is quite confusing right now because they are still so similar for all options. (see comment on `validator`)

Reply:
I think where possible, we should avoid changing APIs in the middle of the program, to avoid confusion like this. So if we start with the parameter `?DEMO=static`, we should use `'static'` throughout if possible. The transformer logic is mostly for type conversions from strings to more intuitive types (such as `'3'` -> `3`).

Reply:
so only the mapping for the `"static"` query parameter value should change (in `ENUMS`). To also keep the transformed value unchanged, as `"static"`.
But the transformer still has to stay in place, because `"false"` has to become `false`